docs/tutorials/connecting-to-the-internet-via-usb.md

```diff
@@ -135,6 +135,99 @@ If you are using an older version, please upgrade.
 
         {% include /util/screenshot.html source="/images/brickman/wired-status-online-connect-automatically-selected.png" %}
 
+
+*   {: tab="linux_command_line"}{% include /style/icon.html type="info" %}
+    These instructions were written using Debian look-alikes for those wanting to know what happens under the hood.
+    The commands need to be executed by root.
+    {:class="alert alert-info"}
+
+    1.  Booting with a standard ev3dev image, lsusb will show
+
+        `Bus 002 Device 008: ID 0694:0005 Lego Group`
+        
+        I'll be using the EV3 as a USB Gadget, as follows.
+
+    2.  When you've copied the ev3dev's image to the SD-card, mount the second partition (EV3DEV_ROOTFS).
+        If your card is /dev/sdc, then mount /dev/sdc2 on a free directory:
+
+        `mount /dev/sdc2 /mnt/sdc2`
+
+    3.  Add the "g_cdc" module to /etc/modules (which currently contains only comments):
+
+        `echo "g_cdc" >> /mnt/sdc2/etc/modules`
+
+        Unmount /dev/sdc2, put the SD-card in the EV3 and start it up.
+
+    4.  Connect the USB cable to your computer and its mini-B plug to the EV3 (slot marked "PC").
+
+    5.  Verify that USB tethering is disabled: *"Wireless and Networks" > "Tethering*.
```

Comment:
Prefer no quotes on `*Wireless and Networks > Tethering*`

If you insist on quotes, you are missing one at the end of `Tethering`.